src/components/SearchBox.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
ah yeah, my initial intention was exporting Bilal's Algolia connected component, instead of copying it. I'm not sure where SearchBar is being used, but I think it's not instantsearch connected.